puppet/modules/airworthy/manifests/install.pp

```diff
@@ -35,4 +35,9 @@
     require => File[$::airworthy::bin_dir],
   }
 
+  file { '/bin/airworthy':
```

Comment:
Why is this necessay? We should not modify /usr/** /bin or any of those package managed directories